qemu/tests/balloon_check.py

```diff
@@ -23,21 +23,25 @@ def __init__(self, test, params, env):
         super(BallooningTest, self).__init__(test, params, env)
 
         self.vm = env.get_vm(params["main_vm"])
-        self.session = self.get_session(self.vm)
-        self.params["balloon_test_setup_ready"] = False
-        if self.params.get('os_type') == 'windows':
-            sleep_time = 180
+        if params.get("paused_after_start_vm") == "yes":
+            self.session = {}
```

Comment:
`self.session` has been initialized in the parent `MemoryBaseTest`, so this should be unnecessary. I suppose it should be:
```
if params.get("paused_after_start_vm") != "yes":
    self.session = self.get_session(self.vm)
    ...
```

Reply:
It is "self.sessions" that is defined in class MemoryBaseTest,  not self.session. Thanks!